Resources/config/yui.yml

```diff
@@ -398,6 +398,7 @@ system:
                         - 'ez-createcontentactionview'
                         - 'ez-movecontentactionview'
                         - 'ez-translateactionview'
+                        - 'ez-translation'
```

Comment:
This should be `ez-translator`. Also the `ez-translator` dependency should be added to all modules that need it (`ez-actionbarview`, `ez-navigationhubview`, ...)